src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
Don't use verbs for class/interface names. The interface name 'AreaCalculator' suggests a utility that calculates area, but the method 'returnInfo' doesn't align with that purpose. Consider renaming the interface to something more appropriate that reflects its responsibility or splitting it if it has multiple responsibilities.